pkg/operator/loglevel/controller.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The OpenShift Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loglevel
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	"github.com/openshift/cloud-credential-operator/pkg/operator/status"
+	"github.com/openshift/cloud-credential-operator/pkg/operator/utils"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	logLevelUtils "github.com/openshift/library-go/pkg/operator/loglevel"
```

Comment:
We prefer to keep cco imports at the end

```suggestion
	operatorv1 "github.com/openshift/api/operator/v1"
	logLevelUtils "github.com/openshift/library-go/pkg/operator/loglevel"
	
	"github.com/openshift/cloud-credential-operator/pkg/operator/status"
	"github.com/openshift/cloud-credential-operator/pkg/operator/utils"
```

Reply:
Sorry, I misunderstood what you were referencing previously. I have resolved this now.